sys/include/net/gcoap.h

```diff
@@ -633,6 +633,20 @@ size_t gcoap_obs_send(const uint8_t *buf, size_t len,
  */
 uint8_t gcoap_op_state(void);
 
+/**
+ * @brief   Add Uri-Query options to a CoAP request
+ *
+ * @param[out] pkt      The package that is being build
```

Comment:
I suggest naming the variable 'pdu', to be consistent with other gcoap functions.